cg/meta/workflow/analysis.py

```diff
@@ -152,6 +152,20 @@ def get_application_type(sample_obj: Sample) -> str:
             return prep_category.lower()
         return AnalysisType.OTHER
 
+    def get_case_application_type(self, case_id: str) -> str:
+        samples: list[Sample] = self.status_db.get_samples_by_case_id(case_id)
+        application_types: list[str] = [self.get_application_type(sample) for sample in samples]
+
+        if not application_types:
+            raise CgError("No application_types found for samples: {samples}")
+
+        return application_types.pop()
```

Comment:
I assume all samples in a case have the same application type? Should we add a check and raise an error if not?

Reply:
Sounds reasonable! I replaced the original if not application_types with this because get_application_type(sample) defaults to returning "other" so it should never activate